CMakeLists.txt

```diff
@@ -95,6 +141,32 @@ if (USE_PACKAGE_MANAGER)
         list(APPEND CMAKE_MODULE_PATH "${CONAN_LIB_DIRS_CATCH2}/cmake/Catch2")
     endif ()
 
+# these compiler options are mostly for debugging and trying to get it to work
```

Comment:
I think these should go in `CompileOptions.cmake`

Reply:
Agreed